### ISSUE TITLE
ClippingRectangleNode inside ScrollView does not work after 3.8rc0

### ISSUE THREAD

**Issue by larsven** [cocos2d/cocos2d-x#13876]

**Comment by larsven** [comment]
to fix it:

```
diff --git a/cocos/2d/CCClippingRectangleNode.cpp b/cocos/2d/CCClippingRectangleNode.cpp
index 98c6b8c..3444261 100644
--- a/cocos/2d/CCClippingRectangleNode.cpp
+++ b/cocos/2d/CCClippingRectangleNode.cpp
@@ -42,11 +42,9 @@ void ClippingRectangleNode::onBeforeVisitScissor()
     if (_clippingEnabled) {
         glEnable(GL_SCISSOR_TEST);

+        const Point pos = convertToWorldSpace(Point(_clippingRegion.origin.x, _clippingRegion.origin.y));
         GLView* glView = Director::getInstance()->getOpenGLView();
-        glView->setScissorInPoints(_clippingRegion.origin.x,
-                                   _clippingRegion.origin.y,
-                                   _clippingRegion.size.width,
-                                   _clippingRegion.size.height);
+        glView->setScissorInPoints(pos.x, pos.y, _clippingRegion.size.width, _clippingRegion.size.height);
     }
 }
```

**Comment by pandamicro** [comment]
It have been fixed by #13913
